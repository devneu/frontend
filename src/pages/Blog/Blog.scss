.container {
  max-width: 1300px;
  margin: 0px auto;

  .sidebar-blog-post {
    margin-left: 20px;
  }
}