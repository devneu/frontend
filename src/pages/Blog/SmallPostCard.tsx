import {Button, Card, Col, Typography} from "antd";
import {ArrowRightOutlined} from "@ant-design/icons";
import React from "react";

const { Title, Text } = Typography;
const { Meta } = Card;

interface SmallPostCard {
    postImg: string,
    postTitle: string,
    postText: string,
    sizeBlock: number
}


const SmallPostCard = ({postImg, postTitle, postText, sizeBlock}: SmallPostCard) => {

    return (
        <Col span={sizeBlock}>
            <Card
                className="post-card"
                cover={
                    <img alt="card-img"
                         src={postImg}
                    />
                }>
                <Meta description="January 1, 2021"/>
                <Title level={2}>{postTitle}</Title>
                <Text className="post-text">{postText}</Text>
                <br/>
                <Button type="primary" className="button-card">Read more<ArrowRightOutlined /></Button>
            </Card>
        </Col>
    );
}

export default SmallPostCard;