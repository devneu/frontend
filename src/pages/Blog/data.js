export const postInfo = [
    {
        "postImg": "https://dummyimage.com/850x350/dee2e6/6c757d.jpg",
        "postTitle": "Featured Post Title",
        "postText": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis aliquid atque, nulla? Quos cum ex quis soluta, a laboriosam. Dicta expedita corporis animi vero voluptate voluptatibus possimus, veniam magni quis!",
        "sizeBlock": 23
    },
    {
        "postImg": "https://dummyimage.com/700x350/dee2e6/6c757d.jpg",
        "postTitle": "Post Title",
        "postText": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis aliquid atque, nulla.",
        "sizeBlock": 11
    },
    {
        "postImg": "https://dummyimage.com/700x350/dee2e6/6c757d.jpg",
        "postTitle": "Post Title",
        "postText": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis aliquid atque, nulla? Quos cum ex quis soluta, a laboriosam.",
        "sizeBlock": 11
    },
    {
        "postImg": "https://dummyimage.com/700x350/dee2e6/6c757d.jpg",
        "postTitle": "Post Title",
        "postText": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis aliquid atque, nulla? Quos cum ex quis soluta, a laboriosam.",
        "sizeBlock": 11
    },
    {
        "postImg": "https://dummyimage.com/700x350/dee2e6/6c757d.jpg",
        "postTitle": "Post Title",
        "postText": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis aliquid atque, nulla? Quos cum ex quis soluta, a laboriosam.",
        "sizeBlock": 11
    }
];