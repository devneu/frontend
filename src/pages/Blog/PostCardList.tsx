import {Col, Row} from "antd";
import {postInfo} from "./data";
import SmallPostCard from "./SmallPostCard";
import React from "react";

const PostCardList  = () => {
    return (
        <Row justify="space-around" align="middle">
            <Col span={14}>
                <Row justify="space-between">
                    {
                        postInfo.map((obj, index) => <Col span={index==0 ? 24: 11}><SmallPostCard key={obj.id} post={obj}/></Col>)
                    }
                </Row>
            </Col>
            <Col span={6}></Col>
        </Row>
    );
}

export default PostCardList;