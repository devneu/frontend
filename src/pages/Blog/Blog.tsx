import React from 'react';
import BlogHeader from '../../components/BlogHeader';
import './Blog.scss';
import PostCardList from '../../components/BlogPosts/PostCardList';
import { postInfo } from '../../components/BlogPosts/data';

const Blog = () => (
  <>
    <BlogHeader title="Post Title" subtitle="Post Subtitle" />
    {/* @ts-ignore */}
    <PostCardList postInfo={postInfo} />
  </>
);

export default Blog;
