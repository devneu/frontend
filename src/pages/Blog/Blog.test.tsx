import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Blog from './Blog';

describe('Blog', () => {
  it('Should successfully render', () => {
    const wrapper = renderer
      .create(
        <BrowserRouter>
          <Blog />
        </BrowserRouter>,
      )
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
