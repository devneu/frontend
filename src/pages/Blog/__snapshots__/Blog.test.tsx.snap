// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog Should successfully render 1`] = `
Array [
  <div
    className="ant-page-header blog-header ant-page-header-ghost"
  >
    <div
      className="ant-page-header-content"
    >
      <h1
        className="ant-typography blog-header__title"
        style={
          Object {
            "WebkitLineClamp": undefined,
          }
        }
      >
        Post Title
      </h1>
      <h3
        className="ant-typography ant-typography-secondary blog-header__subtitle"
        style={
          Object {
            "WebkitLineClamp": undefined,
          }
        }
      >
        Post Subtitle
      </h3>
    </div>
  </div>,
  <div
    className="blog-page container"
  >
    <section
      className="ant-layout ant-layout-has-sider layout"
    >
      <main
        className="ant-layout-content content"
      >
        <div
          className="ant-row"
          style={
            Object {
              "marginBottom": -12,
              "marginLeft": -12,
              "marginRight": -12,
              "marginTop": -12,
            }
          }
        >
          <div
            className="ant-col ant-col-24"
            style={
              Object {
                "paddingBottom": 12,
                "paddingLeft": 12,
                "paddingRight": 12,
                "paddingTop": 12,
              }
            }
          >
            <div
              className="ant-card ant-card-bordered post-feature-card"
            >
              <div
                className="ant-card-cover"
              >
                <img
                  alt="card-img"
                  src="https://dummyimage.com/700x350/dee2e6/6c757d.jpg"
                />
              </div>
              <div
                className="ant-card-body"
                style={Object {}}
              >
                <div
                  className="ant-card-meta"
                >
                  <div
                    className="ant-card-meta-detail"
                  >
                    <div
                      className="ant-card-meta-description"
                    >
                      September 23, 2010
                    </div>
                  </div>
                </div>
                <h2
                  className="ant-typography"
                  style={
                    Object {
                      "WebkitLineClamp": undefined,
                    }
                  }
                >
                  PostPage Title
                </h2>
                <div
                  className="ant-typography post-text"
                  style={
                    Object {
                      "WebkitLineClamp": undefined,
                    }
                  }
                >
                  Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.
                </div>
                <a
                  aria-current={null}
                  href="/blog/1"
                  onClick={[Function]}
                >
                  <button
                    className="ant-btn ant-btn-primary"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Read more
                    </span>
                    <span
                      aria-label="arrow-right"
                      className="anticon anticon-arrow-right"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="arrow-right"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                        />
                      </svg>
                    </span>
                  </button>
                </a>
              </div>
            </div>
          </div>
          <div
            className="ant-col ant-col-12"
            style={
              Object {
                "paddingBottom": 12,
                "paddingLeft": 12,
                "paddingRight": 12,
                "paddingTop": 12,
              }
            }
          >
            <div
              className="ant-card ant-card-bordered post-card"
            >
              <div
                className="ant-card-cover"
              >
                <img
                  alt="card-img"
                  src="https://dummyimage.com/700x350/dee2e6/6c757d.jpg"
                />
              </div>
              <div
                className="ant-card-body"
                style={Object {}}
              >
                <div
                  className="ant-card-meta"
                >
                  <div
                    className="ant-card-meta-detail"
                  >
                    <div
                      className="ant-card-meta-description"
                    >
                      September 23, 2010
                    </div>
                  </div>
                </div>
                <h2
                  className="ant-typography"
                  style={
                    Object {
                      "WebkitLineClamp": undefined,
                    }
                  }
                >
                  PostPage Title
                </h2>
                <div
                  className="ant-typography post-text"
                  style={
                    Object {
                      "WebkitLineClamp": undefined,
                    }
                  }
                >
                  Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.
                </div>
                <a
                  aria-current={null}
                  href="/blog/2"
                  onClick={[Function]}
                >
                  <button
                    className="ant-btn ant-btn-primary"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Read more
                    </span>
                    <span
                      aria-label="arrow-right"
                      className="anticon anticon-arrow-right"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="arrow-right"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                        />
                      </svg>
                    </span>
                  </button>
                </a>
              </div>
            </div>
          </div>
          <div
            className="ant-col ant-col-12"
            style={
              Object {
                "paddingBottom": 12,
                "paddingLeft": 12,
                "paddingRight": 12,
                "paddingTop": 12,
              }
            }
          >
            <div
              className="ant-card ant-card-bordered post-card"
            >
              <div
                className="ant-card-cover"
              >
                <img
                  alt="card-img"
                  src="https://dummyimage.com/700x350/dee2e6/6c757d.jpg"
                />
              </div>
              <div
                className="ant-card-body"
                style={Object {}}
              >
                <div
                  className="ant-card-meta"
                >
                  <div
                    className="ant-card-meta-detail"
                  >
                    <div
                      className="ant-card-meta-description"
                    >
                      September 23, 2010
                    </div>
                  </div>
                </div>
                <h2
                  className="ant-typography"
                  style={
                    Object {
                      "WebkitLineClamp": undefined,
                    }
                  }
                >
                  PostPage Title
                </h2>
                <div
                  className="ant-typography post-text"
                  style={
                    Object {
                      "WebkitLineClamp": undefined,
                    }
                  }
                >
                  Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.
                </div>
                <a
                  aria-current={null}
                  href="/blog/3"
                  onClick={[Function]}
                >
                  <button
                    className="ant-btn ant-btn-primary"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Read more
                    </span>
                    <span
                      aria-label="arrow-right"
                      className="anticon anticon-arrow-right"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="arrow-right"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                        />
                      </svg>
                    </span>
                  </button>
                </a>
              </div>
            </div>
          </div>
          <div
            className="ant-col ant-col-12"
            style={
              Object {
                "paddingBottom": 12,
                "paddingLeft": 12,
                "paddingRight": 12,
                "paddingTop": 12,
              }
            }
          >
            <div
              className="ant-card ant-card-bordered post-card"
            >
              <div
                className="ant-card-cover"
              >
                <img
                  alt="card-img"
                  src="https://dummyimage.com/700x350/dee2e6/6c757d.jpg"
                />
              </div>
              <div
                className="ant-card-body"
                style={Object {}}
              >
                <div
                  className="ant-card-meta"
                >
                  <div
                    className="ant-card-meta-detail"
                  >
                    <div
                      className="ant-card-meta-description"
                    >
                      September 23, 2010
                    </div>
                  </div>
                </div>
                <h2
                  className="ant-typography"
                  style={
                    Object {
                      "WebkitLineClamp": undefined,
                    }
                  }
                >
                  PostPage Title
                </h2>
                <div
                  className="ant-typography post-text"
                  style={
                    Object {
                      "WebkitLineClamp": undefined,
                    }
                  }
                >
                  Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.
                </div>
                <a
                  aria-current={null}
                  href="/blog/4"
                  onClick={[Function]}
                >
                  <button
                    className="ant-btn ant-btn-primary"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Read more
                    </span>
                    <span
                      aria-label="arrow-right"
                      className="anticon anticon-arrow-right"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="arrow-right"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                        />
                      </svg>
                    </span>
                  </button>
                </a>
              </div>
            </div>
          </div>
        </div>
        <ul
          className="ant-pagination pagination"
          style={Object {}}
          unselectable="unselectable"
        >
          <li
            aria-disabled={true}
            className="ant-pagination-prev ant-pagination-disabled"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex={null}
            title="Previous Page"
          >
            <button
              className="ant-pagination-item-link"
              disabled={true}
              tabIndex={-1}
              type="button"
            >
              <span
                aria-label="left"
                className="anticon anticon-left"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="left"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                  />
                </svg>
              </span>
            </button>
          </li>
          <li
            className="ant-pagination-item ant-pagination-item-1 ant-pagination-item-active"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex="0"
            title={1}
          >
            <a
              rel="nofollow"
            >
              1
            </a>
          </li>
          <li
            className="ant-pagination-item ant-pagination-item-2"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex="0"
            title={2}
          >
            <a
              rel="nofollow"
            >
              2
            </a>
          </li>
          <li
            className="ant-pagination-item ant-pagination-item-3"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex="0"
            title={3}
          >
            <a
              rel="nofollow"
            >
              3
            </a>
          </li>
          <li
            className="ant-pagination-item ant-pagination-item-4"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex="0"
            title={4}
          >
            <a
              rel="nofollow"
            >
              4
            </a>
          </li>
          <li
            className="ant-pagination-item ant-pagination-item-5"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex="0"
            title={5}
          >
            <a
              rel="nofollow"
            >
              5
            </a>
          </li>
          <li
            className="ant-pagination-item ant-pagination-item-6"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex="0"
            title={6}
          >
            <a
              rel="nofollow"
            >
              6
            </a>
          </li>
          <li
            aria-disabled={false}
            className="ant-pagination-next"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex={0}
            title="Next Page"
          >
            <button
              className="ant-pagination-item-link"
              disabled={false}
              tabIndex={-1}
              type="button"
            >
              <span
                aria-label="right"
                className="anticon anticon-right"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="right"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                  />
                </svg>
              </span>
            </button>
          </li>
        </ul>
      </main>
      <aside
        className="ant-layout-sider ant-layout-sider-light sidebar"
        style={
          Object {
            "flex": "0 0 400px",
            "maxWidth": "400px",
            "minWidth": "400px",
            "width": "400px",
          }
        }
      >
        <div
          className="ant-layout-sider-children"
        >
          <div
            className="ant-card ant-card-bordered search-widget-card"
          >
            <div
              className="ant-card-head"
              style={Object {}}
            >
              <div
                className="ant-card-head-wrapper"
              >
                <div
                  className="ant-card-head-title"
                >
                  Search
                </div>
              </div>
            </div>
            <div
              className="ant-card-body"
              style={Object {}}
            >
              <form
                className="ant-form ant-form-inline"
                onReset={[Function]}
                onSubmit={[Function]}
              >
                <div
                  className="ant-row ant-form-item"
                  style={Object {}}
                >
                  <div
                    className="ant-col ant-form-item-control"
                    style={Object {}}
                  >
                    <div
                      className="ant-form-item-control-input"
                    >
                      <div
                        className="ant-form-item-control-input-content"
                      >
                        <input
                          className="ant-input input-search"
                          id="searchInput"
                          name="search"
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          placeholder="Enter search term..."
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  className="ant-row ant-form-item"
                  style={Object {}}
                >
                  <div
                    className="ant-col ant-form-item-control"
                    style={Object {}}
                  >
                    <div
                      className="ant-form-item-control-input"
                    >
                      <div
                        className="ant-form-item-control-input-content"
                      >
                        <button
                          className="ant-btn ant-btn-primary button-search"
                          onClick={[Function]}
                          type="submit"
                        >
                          <span>
                            Go!
                          </span>
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </form>
            </div>
          </div>
          <div
            className="ant-card ant-card-bordered categories-widget-card"
          >
            <div
              className="ant-card-head"
              style={Object {}}
            >
              <div
                className="ant-card-head-wrapper"
              >
                <div
                  className="ant-card-head-title"
                >
                  Categories
                </div>
              </div>
            </div>
            <div
              className="ant-card-body"
              style={Object {}}
            >
              <div
                className="ant-row ant-row-start"
                style={
                  Object {
                    "marginBottom": -4,
                    "marginLeft": -4,
                    "marginRight": -4,
                    "marginTop": -4,
                  }
                }
              >
                <div
                  className="ant-col ant-col-12"
                  style={
                    Object {
                      "paddingBottom": 4,
                      "paddingLeft": 4,
                      "paddingRight": 4,
                      "paddingTop": 4,
                    }
                  }
                >
                  <a
                    className="ant-btn ant-btn-link"
                    href="#"
                    onClick={[Function]}
                  >
                    <span>
                      Web Design
                    </span>
                  </a>
                </div>
                <div
                  className="ant-col ant-col-12"
                  style={
                    Object {
                      "paddingBottom": 4,
                      "paddingLeft": 4,
                      "paddingRight": 4,
                      "paddingTop": 4,
                    }
                  }
                >
                  <a
                    className="ant-btn ant-btn-link"
                    href="#"
                    onClick={[Function]}
                  >
                    <span>
                      HTML
                    </span>
                  </a>
                </div>
                <div
                  className="ant-col ant-col-12"
                  style={
                    Object {
                      "paddingBottom": 4,
                      "paddingLeft": 4,
                      "paddingRight": 4,
                      "paddingTop": 4,
                    }
                  }
                >
                  <a
                    className="ant-btn ant-btn-link"
                    href="#"
                    onClick={[Function]}
                  >
                    <span>
                      Freebies
                    </span>
                  </a>
                </div>
                <div
                  className="ant-col ant-col-12"
                  style={
                    Object {
                      "paddingBottom": 4,
                      "paddingLeft": 4,
                      "paddingRight": 4,
                      "paddingTop": 4,
                    }
                  }
                >
                  <a
                    className="ant-btn ant-btn-link"
                    href="#"
                    onClick={[Function]}
                  >
                    <span>
                      JavaScript
                    </span>
                  </a>
                </div>
                <div
                  className="ant-col ant-col-12"
                  style={
                    Object {
                      "paddingBottom": 4,
                      "paddingLeft": 4,
                      "paddingRight": 4,
                      "paddingTop": 4,
                    }
                  }
                >
                  <a
                    className="ant-btn ant-btn-link"
                    href="#"
                    onClick={[Function]}
                  >
                    <span>
                      CSS
                    </span>
                  </a>
                </div>
                <div
                  className="ant-col ant-col-12"
                  style={
                    Object {
                      "paddingBottom": 4,
                      "paddingLeft": 4,
                      "paddingRight": 4,
                      "paddingTop": 4,
                    }
                  }
                >
                  <a
                    className="ant-btn ant-btn-link"
                    href="#"
                    onClick={[Function]}
                  >
                    <span>
                      Tutorials
                    </span>
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            className="ant-card ant-card-bordered side-widget"
          >
            <div
              className="ant-card-head"
              style={Object {}}
            >
              <div
                className="ant-card-head-wrapper"
              >
                <div
                  className="ant-card-head-title"
                >
                  Side Widget
                </div>
              </div>
            </div>
            <div
              className="ant-card-body"
              style={Object {}}
            >
              You can put anything you want inside of these side widgets. They are easy to use, and feature the Bootstrap 5 card component!
            </div>
          </div>
        </div>
      </aside>
    </section>
  </div>,
]
`;
